docs/schema/records_reference.md

```diff
@@ -28,81 +13,112 @@ The versioned release format is designed to support analysis of how data has cha
+
+Release `id` values are only required to be unique within the scope of a contracting process: that is, within the scope of an `ocid` value. As such, a consuming application needs to use that fragment identifier in combination with the `ocid` of the record in order to identify the matching release within the release package.
+
+#### Embedded releases
+
+An embedded release follows the [release schema](reference).
+
+The following example demonstrates the use of embedded releases.
 
+```eval_rst
+.. jsoninclude:: ../examples/record-embedded-releases.json
+   :jsonpointer: /records/0
+   :expand: releases,tag
+   :title: releases
 ```
 
-### Compiled Release
+#### Comparing options
+
+The considerations are:
 
-The compiled release is the latest version of all the data about this contracting process, and has the same schema as a release.
+* Using linked releases yields smaller file sizes than using embedded releases.
+* From a user's perspective, the contents of releases are easier to access as embedded releases than as linked releases (which involves the retrieval process described above).
+* From a publisher's perspective, using linked releases involves publishing release packages and constructing `url` values, whereas using embedded releases doesn't.
 
-The process for creating a compiled release is described in the guidance on [merging](merging).
+### Compiled release
 
-A compiled release provides a snapshot of the current state of a contracting process, and the contents must comply with the release schema, with the exception of any fields on which `"omitWhenMerged": true` is declared.
+The compiled release is the latest version of all the data about a contracting process. In other words, it provides a snapshot of the current state of a contracting process.
 
-### Versioned Release
+The compiled release follows the [release schema](reference), with the exception of any fields on which `"omitWhenMerged": true` is declared.
 
-A versioned release contains the history of all the data in the compiled release, including which fields have changed, when they were changed, and the release which updated them.
+The process for creating a compiled release from individual releases is described in the [merging](merging) reference.
 
-This versioned information is valuable for many use cases relating to contract monitoring. However, it can add substantially to file sizes, and so where publishers do generate this at source, they can consider publishing versioned and non-versioned copies of the records.
+### Versioned release
 
-Third parties can use the information in the releases list to fetch source data, compile and verify their own version history for a contract.
+A versioned release contains the history of all the data in the compiled release, including which fields were changed, when they were changed, and the release in which they were changed.
```

Comment:
maybe add "..including the fields first value, which fields were changed.." as the versionedRelease doesn't include only the fields that changed but all of them

Reply:
I changed the paragraph to:

> A versioned release aggregates all values of all fields from all releases. For each field, it describes the history of values from the initial value to the current value, including the `id`, `date` and `tag` of the release in which it was changed.

Reply:
perfect!